oozie/oozie.sh

```diff
@@ -48,8 +49,30 @@ function install_oozie() {
   master_node=$(/usr/share/google/get_metadata_value attributes/dataproc-master)
 
   # Upgrade the repository and install Oozie
-  retry_apt_command "apt-get update"
-  retry_apt_command "apt-get install -q -y oozie oozie-client"
+  if [[ "$DATAPROC_IMAGE" == *"cen"* ]]; then
```

Comment:
Use `if command -v apt-get >/dev/null; then` check for consistency with other scripts.